src/async-wrap.cc

```diff
@@ -791,3 +792,56 @@ void EmitAsyncDestroy(Isolate* isolate, async_context asyncContext) {
 }  // namespace node
 
 NODE_MODULE_CONTEXT_AWARE_BUILTIN(async_wrap, node::AsyncWrap::Initialize)
+
+
+// Only legacy public API below this line.
+
+namespace node {
+
+MaybeLocal<Value> MakeCallback(Isolate* isolate,
```

Comment:
All `MakeCallback` functions are in `node.cc` including the legacy ones. But since this won't land in master I don't care that much.